simplefs/simplefs.go

```diff
@@ -517,6 +542,20 @@ func (k *SimpleFS) SimpleFSClose(ctx context.Context, opid keybase1.OpID) (err e
 	return err
 }
 
+// SimpleFSCancel starts to cancel op with the given opid.
+// Also remove any pending references of opid everywhere.
+func (k *SimpleFS) SimpleFSCancel(_ context.Context, opid keybase1.OpID) {
```

Comment:
The generated protocol is going to expect a method that returns `error` I think.